config/settings/dev.py

```diff
@@ -32,7 +24,7 @@
 INTERNAL_IPS = ["127.0.0.1", "10.0.2.2"]
 
 # Add in INTERNAL_IPS internal IP of the docker container if DEGUG == True.
-if env("DEBUG"):
+if env("DJANGO_DEBUG"):
```

Comment:
По идее, можно использовать такую конструкцию
```suggestion
if DEBUG:
```

Reply:
Другой вопрос, что на стейдже у нас всегда включён дебаг, нужно ли тут делать эту проверку?

Reply:
Ну вот и я над этим думал, что по факту здесь `if` условие, которое всегда выполняется. Можем оставить на случай, если вдруг на стэйдже все же может возникнуть сценарий `DEBUG = False`?

Reply:
Ну, давай останется,  ничего тут страшного нет